base_layer/wallet/src/transaction_service/service.rs

```diff
@@ -2733,8 +2767,16 @@ where
     /// Handle an incoming basenode response message
     pub async fn handle_base_node_response(
         &mut self,
-        response: base_node_proto::BaseNodeServiceResponse,
+        response: Result<base_node_proto::BaseNodeServiceResponse, prost::DecodeError>,
     ) -> Result<(), TransactionServiceError> {
+        if let Err(e) = response {
+            // TODO: Should we switch base nodes?
```

Comment:
```suggestion
            //Should we switch base nodes?
```

created an issue: https://github.com/tari-project/tari/issues/5695